torch/_inductor/codegen/cpp.py

```diff
@@ -2606,7 +2606,11 @@ def indirect_assert(self, var, lower, upper, mask=None):
             assert upper
             cond = f"{var} < {upper}"
             cond_print = f"{var} < {upper_scalar}"
-        cond = f"({self._get_mask_type(var.dtype)}({cond})).all_masked()"
+        cond = f"({self._get_mask_type(var.dtype)}({cond}))"
+        if self._load_mask is not None:
```

Comment:
```suggestion
        if mask:
```
Let's mirror what we do in the general case at
https://github.com/pytorch/pytorch/blob/312652c3258a3a8fec8fbfe6a9e8887e23d39c13/torch/_inductor/codegen/common.py#L1754-L1755
and just make sure we pass `self._load_mask` to this function when needed.